test/signs_ui/config/expiration_test.exs

```diff
@@ -50,9 +51,141 @@ defmodule SignsUi.Config.ExpirationTest do
         }
       }
 
-      assert SignsUi.Config.Expiration.expire_signs(state, fn ->
-               DateTime.new!(~D[2019-01-15], ~T[08:00:00], "America/New_York")
-             end) == expected_updates
+      assert SignsUi.Config.Expiration.expire_signs(
+               state,
+               fn -> DateTime.new!(~D[2019-01-15], ~T[08:00:00], "America/New_York") end,
+               fn -> MapSet.new([]) end
+             ) == expected_updates
+    end
+  end
+
+  describe "expire_signs_via_alert/2" do
+    test "produces correct updates when alerts expire" do
+      alert_state = %SignsUi.Alerts.State{
+        alerts: %{
+          "Red" => %{
+            "1234" => %{},
+            "5678" => %{}
+          },
+          "Blue" => %{
+            "abc" => %{}
+          }
+        }
+      }
+
+      alert_ids = SignsUi.Config.Expiration.active_alert_ids(alert_state)
```

Comment:
This is from before. You don't need this part of the test anymore since the code was moved to the other module.